src/client/utils/hooks/useQuoteCartData.ts

```diff
@@ -1,21 +1,222 @@
+            : 'CHECKOUT_DETAILS_NUMBER_OF_BATHROOMS_SUFFIX_ONE',
+      },
+    },
+    {
+      label: 'CHECKOUT_DETAILS_YEAR_OF_CONSTRUCTION',
+      value: { value: data.yearOfConstruction },
+    },
+    {
+      label: 'CHECKOUT_DETAILS_IS_PARTLY_SUBLET',
+      value: {
+        textKey: data.isSubleted ? 'YES' : 'NO',
+      },
+    },
+  ]
+}
+
+const getExtraBuildingsDetails = (quoteDetails: GenericQuoteData) => {
+  if (!quoteDetails.extraBuildings) {
+    return []
+  }
+  const extraBuildings = quoteDetails.extraBuildings.map(
+    (extraBuilding: any) => [
+      {
+        label: 'CHECKOUT_DETAILS_EXTRA_BUILDINGS_BUILDING_TYPE',
+        value: { value: extraBuilding.displayName },
+      },
+      {
+        label: 'CHECKOUT_DETAILS_EXTRA_BUILDINGS_SIZE',
+        value: {
+          value: extraBuilding.area,
+          suffix: 'CHECKOUT_DETAILS_SQM_VALUE',
+        },
+      },
+      {
+        label: 'CHECKOUT_DETAILS_EXTRA_BUILDINGS_HAS_WATER_CONNECTED',
+        value: { textKey: extraBuilding.hasWaterConnected ? 'YES' : 'NO' },
+      },
+    ],
+  )
+  return extraBuildings
+}
+
+const getPeopleDetails = (quoteDetails: GenericQuoteData) => {
+  const { numberCoInsured } = quoteDetails
+  const householdSize = numberCoInsured + 1
+  return [
+    {
+      label: 'CHECKOUT_DETAILS_HOUSEHOLD_SIZE',
+      value: {
+        value: householdSize,
+        suffix:
+          householdSize > 1
+            ? 'CHECKOUT_DETAILS_NUMBER_OF_PEOPLE_SUFFIX_MANY'
+            : 'CHECKOUT_DETAILS_NUMBER_OF_PEOPLE_SUFFIX_ONE',
+      },
+    },
+    ...getStudentOrYouth(quoteDetails),
+  ]
+}
+
+const getStudentOrYouth = (data: GenericQuoteData) => {
+  if (data.isStudent) {
+    return [{ label: '+', value: { textKey: 'CHECKOUT_DETAILS_STUDENT' } }]
+  }
+  if (data.isYouth) {
+    return [{ label: '+', value: { textKey: 'CHECKOUT_DETAILS_YOUTH' } }]
+  }
+
+  return []
+}
 
 export const useQuoteCartData = () => {
   const { id: quoteCartId } = useParams<{ id: string }>()
   const { isoLocale } = useCurrentLocale()
-
   const { data, loading, error } = useQuoteCartQuery({
-    variables: {
-      id: quoteCartId,
-      locale: isoLocale,
-    },
+    variables: { id: quoteCartId, locale: isoLocale },
+  })
+
+  const [selectedInsuranceTypes] = useSelectedInsuranceTypes()
+  const selectedQuoteBundle = getSelectedBundleVariant(
+    data,
+    selectedInsuranceTypes,
+  )
+  if (!selectedQuoteBundle) return null
+
+  const prices = selectedQuoteBundle?.bundle.quotes.map((item) => {
+    return { displayName: item.displayName, price: item.price.amount }
   })
 
+  const mainQuote = getMainQuote(selectedQuoteBundle.bundle)
+
+  const priceData = {
+    prices: prices,
+    discount: getDiscountAmount(selectedQuoteBundle.bundle),
+    currency: getBundleCurrency(selectedQuoteBundle.bundle),
+    totalBundleCost: getTotalBundleCost(selectedQuoteBundle.bundle),
+    campaignName: data?.quoteCart.campaign?.displayValue,
```

Comment:
There is already a selector that gets you the campaign from the query under `api/quoteCartQuerySelectors.ts`

So something like this should work
```suggestion
    campaignName: getCampaign(data)?.displayValue,
```

Reply:
that's great 👍